test/TrieTest.mo

```diff
@@ -0,0 +1,38 @@
+import Trie "mo:base/Trie";
```

Comment:
It would be nice to add alot more test here. The matchers library should be good enough. There's some examples in other tests (i.e. testText.mo)

Reply:
I agree, but that sounds like a worthwhile effort for a follow-up PR that focuses on expanding the tests, which I agree are lacking.

Really, the "test" I have here is just to test the example that I provided in the doc; it's not meant to serve as what you are asking about now.  I will rename this file to TrieExamples to make that clearer and plan to do a follow up, using matchers and the other tests as examples, per your advice.

Reply:
Ok, that makes sense.